Clockify/ToggleAction.cs

```diff
@@ -93,14 +92,13 @@ public override void ReceivedGlobalSettings(ReceivedGlobalSettingsPayload payloa
         {
         }
 
-        private string CreateTimerText()
+        private string CreateTimerText(string timerTime)
         {
-            if (string.IsNullOrEmpty(_settings.TimeName))
-            {
-                return string.IsNullOrEmpty(_settings.TaskName) ? $"{_settings.ProjectName}" : $"{_settings.ProjectName}:\n{_settings.TaskName}";
-            }
-
-            return $"{_settings.TimeName}";
+            return _settings.TitleFormat
+                .Replace("{projectName}", _settings.ProjectName)
+                .Replace("{taskName}", _settings.TaskName)
+                .Replace("{timerName}", _settings.TimerName)
+                .Replace("{timer}", timerTime);
```

Comment:
I suggest to keep the original behavior and only change it `TitleFormat` is not empty